tests/trace/test_b12x_moe_trace.py

```
@@ -69,6 +73,14 @@ def test_b12x_reference_uses_activation_precision_and_fc2_scale():
             activation_precision="fp4",
         )
 
+    with pytest.raises(ValueError, match="compressed_tensors.*w4a16"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)